### PR TITLE
feat: OWL-118 handle repetitions

### PR DESCRIPTION
**Pull request by healukas**

### CODE REVIEW

File: Healex.HL7v2Anonymizer.Tests/AnonymizerTests.cs [hunk 1]
```diff
@@ -1,108 +1,337 @@
...
         }
+        #endregion
 
-        private static ReplacementOptions GetReplacementOptions() {
-            var builder = new ConfigurationBuilder()
-                .SetBasePath(Directory.GetCurrentDirectory())
-                .AddJsonFile("appsettings.json", optional: false);
+        #region TestData
+        
+        private static Message CreateDefaultTestMessage(int repetitionsPid5)
```
Comment (msdoege) [inline Healex.HL7v2Anonymizer.Tests/AnonymizerTests.cs:129]: Could have added a default value of 0 for the `repetitionsPid5`

File: Healex.HL7v2Anonymizer.Tests/AnonymizerTests.cs [hunk 2]
```diff
@@ -1,108 +1,337 @@
...
+            TestAnonymization(CreateDefaultTestMessage(0));
         }
 
         [TestMethod]
-        [DeploymentItem("Healex.HL7v2Anonymizer.Tests/TestData", "TestData")]
-        public void AnonymizerTestAdtSegmentOrderWithAnonymization() {
+        public void AnonymizerTestOru()
+        {
+            TestAnonymization(CreateDefaultTestMessage(0));
+        }
```
Comment (msdoege) [inline Healex.HL7v2Anonymizer.Tests/AnonymizerTests.cs:54]: After removing the ORU test message, this is test is redundant as it is exactly the same as `AnonymizerTestAdt`.

File: Healex.HL7v2Anonymizer/Services/Anonymizer.cs [hunk 3]
```diff
@@ -42,9 +46,45 @@ public class Anonymizer {
 
         #region Auxiliar Methods
 
+        public List<Replacement> ExpandReplacementRules(Segment segment, Replacement[] originalRules)
```
Comment (msdoege) [inline Healex.HL7v2Anonymizer/Services/Anonymizer.cs:49]: Please consider pointing out, that only rules for repeated fields are expanded here. You could included it in the name like `ExpandReplacementRulesForRepeatedFields` or add a comment.

**Review by msdoege — APPROVED** [review]
Well done. Only some comments for your consideration, nothing mission critical though.

Could you please add a sentence to the readme that if a field is repeated, all instances of that field are replaced with a matching replacement. For me that would be the expected behaviour anyway, but it wouldn't hurt to be explicit about it.

File: Healex.HL7v2Anonymizer/Services/Anonymizer.cs (hunk 3 above)
Comment (healukas) [reply Healex.HL7v2Anonymizer/Services/Anonymizer.cs:49]: I like that. Done

**Pull request merged**